### PR TITLE
dev mode for new source connection shows the API call to generate source connect link, not the iframe info

### PR DESCRIPTION
**Pull request by LemonFace0309**
CleanShot 2024-05-24 at 15 36 44@2x

### CODE REVIEW

**Comment by LemonFace0309** [comment]
* **#41** <a href="https://app.graphite.dev/github/pr/sutrolabs/census-embedded-demo/41?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @LemonFace0309 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: pages/integrations/import-dataset.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function ImportDataset({
       <Head>
         <title>Data Sources - Integrations - Census Embedded Demo App</title>
       </Head>
-      <div className="flex flex-col gap-2 lg:flex-row lg:items-center lg:justify-between">
+      <div className="flex flex-col gap-2 xl:flex-row xl:items-center xl:justify-between">
```
Comment (LemonFace0309) [inline pages/integrations/import-dataset.js:32]: small ui improvement since lg was overcrowding

File: components/RequestTooltip.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Tooltip } from "react-tooltip"
 
 export default function RequestTooltip({ anchorSelect, url, method, headers, body, devMode }) {
   return (
-    <Tooltip anchorSelect={anchorSelect} className={devMode ? "" : "hidden"}>
+    <Tooltip anchorSelect={anchorSelect} className={devMode ? "z-10" : "z-10 hidden"}>
```
Comment (LemonFace0309) [inline components/RequestTooltip.js:5]: was conflicting with the text for some buttons before this change

File: components/Source.js [hunk 3]
```diff
@@ -212,20 +211,37 @@ export default function Source({
...
+                2,
+              )}
+            </pre>
+          }
+          body={
+            <pre>
+              {JSON.stringify(
+                {
+                  type,
+                  redirect_uri: window.location.href,
```
Comment (yeldah3) [inline components/Source.js:237]: Can we only include this request param if in redirect mode (not embed mode)?

**Review by yeldah3 — COMMENTED** [review]
This is sweet, thanks Charles! While you're here can we also add a similar tooltip for creating sync management links when opening the new sync experience?

**Review by yeldah3 — APPROVED** [review]

**Comment by LemonFace0309** [comment]
> This is sweet, thanks Charles! While you're here can we also add a similar tooltip for creating sync management links when opening the new sync experience?

Yep! That was merged in in another pr :)

https://github.com/sutrolabs/census-embedded-demo/pull/47
CleanShot 2024-05-26 at 23 43 25@2x

**Pull request merged**